code/go/0chain.net/smartcontract/minersc/miner_test.go

```diff
@@ -0,0 +1,298 @@
+package minersc
+
+import (
+	"testing"
+
+	"0chain.net/chaincore/block"
+	"0chain.net/chaincore/state"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const stakeVal, stakeHolders = 1e10, 3
+
+func Test_DeleteNode(t *testing.T) {
+
+	var (
+		balances *testBalances
+		msc      = newTestMinerSC()
+		now      int64
+
+		miners   []*miner
+		sharders []*sharder
+	)
+
+	t.Run("delete miner pending pool", func(t *testing.T) {
```

Comment:
For consistancy can now we adopt the [table driven subtest style](https://blog.golang.org/subtests#TOC_4.) of @MurashovVen's unit tests.